Packs/XSOARmirroring/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### XSOAR Mirroring
+- Maintenance and stability enhancements.
```

Comment:
Lets elaborate. We removed few incident types, mapping, classifier.

Reply:
Will go over it with @kirbles19

Reply:
@anara123 What was removed exactly?

Reply:
Removed:
- Incident field (frompong)
- Incident Types (Ping and Pong)
- Classifier
- Mapper out

All these are considered breaking changes.

Reply:
But this is from the pack in general, not specific to the integration, right?
What should the RN for the integration itself be?

Reply:
Also, I think users will want a bit more information about what exactly this breaks and what the workaround is.